Dockerfile

```diff
@@ -1,21 +1,19 @@
 FROM ubuntu:xenial
-WORKDIR /home/dev
 
 RUN apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install -y \
+    apt-get install -y --no-install-recommends software-properties-common && \
```

Comment:
@csboling I made some changes to the Dockerfile to make it a bit quicker since there is no layer caching on GitHub Actions (yet).

Difference in image size (top is after the changes)
```
$ d images | grep -i crow
crow-builder                           small               5ca08df689fc        About an hour ago   835MB
crow-builder                           latest              a5ae1f93f8a9        About an hour ago    923MB
```
It seems to be building the firmware correctly, but would probably be good to test if it still works as expected for you?